.github/workflows/publish.yml

```
@@ -0,0 +1,25 @@
+name: publish
+
+on:
+  release:
+    types: [created]
```

The events here are a bit confusing, it seems from [the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) that `created` does not trigger for draft releases, so is possibly the same as `published`. But still wondering if `published` is more obvious, as its in line with the UI where you "publish release" or "save draft".

```suggestion
    types: [published]
```